src/main/scala/geotrellis/sdg/FilteredRoadsPipeline.scala

```diff
@@ -0,0 +1,38 @@
+package geotrellis.sdg
+
+import java.net.URI
+
+import geotrellis.layer.LayoutLevel
+import geotrellis.vector._
+import geotrellis.vectortile.{VBool, VString}
+import org.apache.spark.sql.{DataFrame, Row}
+import vectorpipe.vectortile.{LayerMultiplicity, Pipeline, SingleLayer, VectorTileFeature}
+
+case class FilteredRoadsPipeline(geometryColumn: String,
+                                 baseOutputURI: URI,
+                                 reduceToIncludedZoom: Int) extends Pipeline {
+
+  override val layerMultiplicity: LayerMultiplicity = SingleLayer("roads")
+
+  override def reduce(input: DataFrame, layoutLevel: LayoutLevel, keyColumn: String): DataFrame = {
+    if (reduceToIncludedZoom == layoutLevel.zoom) {
+      input.where("isIncluded = true")
+    } else {
+      input
+    }
+  }
+
+  override def pack(row: Row, zoom: Int): VectorTileFeature[Geometry] = {
+    val geom = row.getAs[Geometry]("geom")
+    val highway = row.getAs[String]("highway")
+    val surface = row.getAs[String]("surface")
+    val isIncluded = row.getAs[Boolean]("isIncluded")
+
+    Feature(geom, Map(
+      "highway" -> VString(highway),
+      "surface" -> VString(surface),
+      "isIncluded" -> VBool(isIncluded)
+    ))
```

Comment:
A thought was that including the feature OSM ID would be a valuable way for somebody to click over to OSM editor and change the tags for the future they felt was not being correctly tagged.

Given TurboP mode I'm not asking for a change, just pointing out possible future feature.

Reply:
Trival to add, so I did.